app/src/testDemo/kotlin/com/google/samples/apps/nowinandroid/ui/NiaAppScreenSizesScreenshotTests.kt

```diff
@@ -135,27 +137,31 @@ class NiaAppScreenSizesScreenshotTests {
     }
 
     private fun testNiaAppScreenshotWithSize(width: Dp, height: Dp, screenshotName: String) {
+        // TODO: Remove this when https://b.corp.google.com/issues/327149166 is fixed
```

Comment:
Avoiding the corp link is nicer for everyone externally when it is a public issue.

```suggestion
        // TODO: Remove this when https://issuetracker.google.com/issues/327149166 is fixed
```